superbench/benchmarks/return_code.py

```diff
@@ -29,6 +29,7 @@ class ReturnCode(Enum):
     MICROBENCHMARK_RESULT_PARSING_FAILURE = 33
     MICROBENCHMARK_UNSUPPORTED_ARCHITECTURE = 34
     MICROBENCHMARK_DEVICE_GETTING_FAILURE = 35
+    MPI_INIT_FAILURE = 36
```

Comment:
Please follow the naming method as before. It's should be MICROBENCHMARK_MPI_INIT_FAILURE.